src/checkers/inference/DefaultSlotManager.java

```diff
@@ -601,7 +598,7 @@ private TypeKind getArithmeticResultKind(AnnotatedTypeMirror lhsAtm, AnnotatedTy
 
     @Override
     public ArithmeticVariableSlot createArithmeticVariableSlot(
-            AnnotationLocation location, AnnotatedTypeMirror lhsAtm, AnnotatedTypeMirror rhsAtm) {
+            AnnotationLocation location, TypeMirror lhs, TypeMirror rhs) {
```

Comment:
Why are there no calls to this method that need to be changed?

Reply:
Because `ArithmeticVariableSlot`s are only created in the value-inference downstream